ios/RNLinksdk.m

```diff
@@ -99,14 +99,16 @@ - (void)stopObserving {
     NSString *paymentTokenInput = [RCTConvert NSString:configuration[kRNLinkKitConfigPaymentTokenKey]];
     NSString *oauthStateId = [RCTConvert NSString:configuration[kRNLinkKitConfigOAuthStateIdKey]];
     NSString *institution = [RCTConvert NSString:configuration[kRNLinkKitConfigInstitutionKey]];
+    NSString *publicKey = [RCTConvert NSString:configuration[kRNLinkKitConfigInstitutionKey]];
```

Comment:
Should be `kRNLinkKitConfigPublicKeyKey` instead of `kRNLinkKitConfigInstitutionKey`

Reply:
My bad, thank you!